build-tools/packages/build-cli/src/BasePackageCommand.ts

```diff
@@ -178,6 +178,162 @@ export abstract class PackageCommand<
 	}
 }
 
+export abstract class PackageCommandWithBuildProject<
```

Comment:
Is this used? If not, I say remove it for now. If we need it now, then we'll need to update this implementation to use the filtering/selection logic from build-infra.